src/test/transaction_tests.cpp

```diff
@@ -475,6 +475,7 @@ BOOST_AUTO_TEST_CASE(test_simple_joinsplit_invalidity)
         jsdesc->nullifiers[1] = GetRandHash();
 
         newTx.vjoinsplit.push_back(JSDescription());
+        jsdesc = &newTx.vjoinsplit[0];
```

Comment:
Non-blocking: Could maybe use a comment pointing out why this is necessary, although someone can always look at `git blame` to figure it out if they can't see it immediately.